apps/builder/app/builder/features/style-panel/shared/property-name.tsx

```diff
@@ -57,6 +62,24 @@ const PropertyPopoverContent = ({
           {properties.map((property) => {
             const styleValueInfo = style[property];
 
+            if (styleValueInfo?.previousSource) {
+              const { value, styleSourceId } = styleValueInfo.previousSource;
+              const styleSource = styleSources.get(styleSourceId);
+              let name: undefined | string = undefined;
+              if (styleSource?.type === "local") {
+                name = "local style source";
```

Comment:
it would be great to extract all the primitive logic for the text into a function which we would be able to unit test easily without the component and stores stuff

Reply:
Let's do this once refactor to new design